### PR TITLE
kbds: add layout info for u-d-i

### PR DESCRIPTION
**Pull request by dbungert**

### CODE REVIEW

File: kbds/keyboard-configuration.yaml [hunk 1]
```diff
@@ -1,101 +1,248 @@
 # File generated by ./scripts/keyboard-scraper.sh, please do not edit manually
```
Comment (ogayot) [inline kbds/keyboard-configuration.yaml:1]: Should we update/drop this line?

**Review by ogayot — APPROVED** [review]
LGTM
One minor observation

File: kbds/keyboard-configuration.yaml (hunk 1 above)
Comment (dbungert) [reply kbds/keyboard-configuration.yaml:1]: Yes, it's slightly incorrect now with
* more input data
* passing through pyyaml for normalization and merging

I will leave for now, as we are considering generating at build time.  It's still mostly correct.

**Comment by dbungert** [comment]
@jpnurmi merged

File: kbds/keyboard-configuration.yaml (hunk 1 above)
Comment (jpnurmi) [reply kbds/keyboard-configuration.yaml:1]: Before I forget, this is how I constructed the full list of locales from the .arb files at https://github.com/canonical/ubuntu-flutter-plugins/tree/main/packages/ubuntu_localizations/lib/src/l10n

```bash
for arb in *.arb; do
    lang=$(jq -r '.["@@locale"]' $arb | cut -d_ -f1)
    country=$(jq -r '.["countryCode"]' $arb)
    echo "${lang}_${country}.UTF-8"
done
```

**Pull request merged**